### PR TITLE
Security attire revamp & trench-coat resprite

### PR DESCRIPTION
**Pull request by AmoryBlaine**
- Removes armor values to all security jackets and trench-coats
- Replaces old security jackets with new versions, adds new investigative jacket in green.
- Replaces old trench-coats with new versions, adds more variation.
- Trench-coats are no longer investigative attire and are available to everyone.
- Removed excess from security lockers. (Removed commissar outfit, redundant gloves, redundant shoes, classic detective outfit, forensic jackets)
- Removed old sec beret from locker and loadout, added proper beret to loadout.
- Resprited DEFAULT labcoat.
- Resprited straight jacket.

### CODE REVIEW

**Comment by ParadoxSpace** [comment]
I would really rather you leave the armor values as-is.

**Comment by AmoryBlaine** [comment]
> 
> 
> I would really rather you leave the armor values as-is.

I am directly opposed to this, but will comply, if ordered to by a dev. I really do believe that if you're armored, you should be wearing just that; armor. It's unfair and confusing to the average person to have sec's jackets just absorb damage because they're sec.

**Comment by AmoryBlaine** [comment]
!review

**Comment by AmoryBlaine** [comment]
If Detectives are to have armored coats, I think that they SHOULD be the same sprites as the rest. Yes, this is contrary to what I said before, but I feel that there's literally no point in making a unique sprite for armored trench-coats, since the whole point of an armored coat is to be inconspicuous.

File: code/modules/clothing/suits/jobs.dm [hunk 1]
```diff
@@ -96,102 +96,41 @@ obj/item/clothing/suit/apron/overalls/blue
...
-	blood_overlay_type = "coat"
-	icon_open = "detective_open"
-	icon_closed = "detective"
-	body_parts_covered = UPPER_TORSO|ARMS
-	allowed = list(/obj/item/tank/emergency_oxygen, /obj/item/device/flashlight,/obj/item/gun/energy,/obj/item/gun/projectile,/obj/item/ammo_magazine,/obj/item/ammo_casing,/obj/item/melee/baton,/obj/item/handcuffs,/obj/item/storage/fancy/cigarettes,/obj/item/flame/lighter,/obj/item/device/taperecorder)
-	armor = list(melee = 50, bullet = 10, laser = 25, energy = 10, bomb = 0, bio = 0, rad = 0)
-	sprite_sheets = list("Vox" = 'icons/mob/species/vox/suit.dmi')
-	siemens_coefficient = 0.7
-
-/obj/item/clothing/suit/storage/toggle/det_trench/black
```
Comment (Doxxmedearly) [inline code/modules/clothing/suits/jobs.dm]: This is still mapped in. Will have to update map to appease Travis if you're deleting this path. Also don't forget to map in any replacements.

Comment (AmoryBlaine) [reply code/modules/clothing/suits/jobs.dm]: How do I find out where it's mapped in? I'm not sure how to remove it, or replace it.

Comment (Doxxmedearly) [reply code/modules/clothing/suits/jobs.dm]: I don't THINK Travis gives an exact location. It just gave me: _Invalid type in map. /obj/item/clothing/suit/storage/toggle/det_trench/black_

I'd probably check the locker in the detective's officer as my first guess. Unfortunately I'm not too familiar with the mapping tools so I'm not sure if there's a way to search item paths in there. Hopefully someone more familiar can help.

Comment (AmoryBlaine) [reply code/modules/clothing/suits/jobs.dm]: Alright, thank you. I'll ask around.

**Comment by alsoandanswer** [comment]
Alright go into Dreammaker and search "aurora"

Tick all the .dmm maps with "aurora"

Compile. It should tell you which levels have problems.

Open them up by double clicking the problem .dmm files.

It will prompt you if you've modified the path for toggle/det_trench/black or whatever. You can fix it by putting in the new path. No mapping experience required.

File: code/modules/clothing/suits/jobs.dm (hunk 1 above)
Comment (AmoryBlaine) [reply code/modules/clothing/suits/jobs.dm]: I figured it out, and removed it. The placement was for an easter egg with skeletons in the walls. So, no replacement.

**Comment by AmoryBlaine** [comment]
> 
> 
> Alright go into Dreammaker and search "aurora"
> 
> Tick all the .dmm maps with "aurora"
> 
> Compile. It should tell you which levels have problems.
> 
> Open them up by double clicking the problem .dmm files.
> 
> It will prompt you if you've modified the path for toggle/det_trench/black or whatever. You can fix it by putting in the new path. No mapping experience required.

ye, I figured it out, thank you though.

**Comment by alsoandanswer** [comment]
I think you should do it again

As of now you have 79,344 deletions

**Comment by AmoryBlaine** [comment]
!wip

**Pull request closed without merging**